nixos/modules/services/networking/pixiecore.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pixiecore;
+
+in
+
+{
+
+  options = {
+
+    services.pixiecore = {
+
+      enable = mkEnableOption "Pixiecore";
+
+      debug = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Log more things that aren't directly related to booting a recognized client";
+      };
+
+      logTimestamps = mkOption {
```

Comment:
If this is logging to syslog then won't time stamps always be added anyways? If this is true will there be 2 timestamps for every entry?